src/tools/cephfs/cephfs-shell

```diff
@@ -94,13 +94,18 @@ def to_bytes(string):
     return bytes(string, encoding='utf-8')
 
 
-def list_items(dir_name=''):
+def list_items(shell, dir_name=''):
+    d = None
     if not isinstance(dir_name, bytes):
         dir_name = to_bytes(dir_name)
     if dir_name == '':
         d = cephfs.opendir(cephfs.getcwd())
     else:
-        d = cephfs.opendir(dir_name)
+        try:
+             d = cephfs.opendir(dir_name)
+        except:
+            dir_name = dir_name.decode('utf-8')
```

Comment:
why?

Reply:
if the user enters an invalid directory name, then an exception is thrown when attempting to open it